syntax = "proto3";

package chatgpt;

option go_package = "github.com/onepiece010938/go-line-message-analyzer/internal/adapter/grpc/chatgpt";

message ChatGptRequest{
    string role = 1;
    string content = 2;

}

message ChatGptReply{
    string response = 1;
}

// original
// syntax = "proto3";

// message ChatGptRequest{
//     string role = 1;
//     string content = 2;
// }

// message ChatGptReply{
//     string response = 1;
// }